src/Http/Http.Abstractions/src/PublicAPI.Unshipped.txt

```diff
@@ -18,6 +18,7 @@ Microsoft.AspNetCore.Http.Metadata.IFromServiceMetadata
 Microsoft.AspNetCore.Http.Endpoint.Endpoint(Microsoft.AspNetCore.Http.RequestDelegate? requestDelegate, Microsoft.AspNetCore.Http.EndpointMetadataCollection? metadata, string? displayName) -> void
 Microsoft.AspNetCore.Http.Endpoint.RequestDelegate.get -> Microsoft.AspNetCore.Http.RequestDelegate?
 Microsoft.AspNetCore.Routing.RouteValueDictionary.TryAdd(string! key, object? value) -> bool
+abstract Microsoft.AspNetCore.Http.HttpRequest.ContentType.get -> string?
```

Comment:
Also add a `*REMOVED*` entry above for the old signature.

Reply:
*Removed* entry has been added to Unshipped.txt.  I attempted to update the DefaultHttpRequest, and that seemed to go fine.  I am not sure how you want to address the null possibility in the HttpLoggingMiddleware class.

Reply:
`ContentType` comes from a `StringValues` so `get` can never return null?

Reply:
> `ContentType` comes from a `StringValues` so `get` can never return null?

The default value of a StringValues implicitly converts to null. Right?
https://github.com/dotnet/runtime/blob/41f3d48b520d4cf4f9e051dc07f480dcb9c62e08/src/libraries/Microsoft.Extensions.Primitives/src/StringValues.cs#L177-L191

Reply:
Which is a different value than calling `.ToString()` on it... interesting type 😅 https://github.com/dotnet/runtime/blob/41f3d48b520d4cf4f9e051dc07f480dcb9c62e08/src/libraries/Microsoft.Extensions.Primitives/src/StringValues.cs#L168-L171